corehq/apps/enterprise/enterprise.py

```diff
@@ -383,3 +389,85 @@ def _get_individual_export_rows(self, exports, export_line_counts):
             )
 
         return rows
+
+
+class EnterpriseSMSReport(EnterpriseReport):
+    title = gettext_lazy('SMS Sent')
```

Comment:
Change the title to align with the design doc

Reply:
Addressed in fc4fccdf5a1f35f682deee56395310c127614c83